### PR TITLE
TICKET-62-RELEASE

### PR DESCRIPTION
**Pull request by Kushalbaj**

### CODE REVIEW

File: src/components/BlogPost.jsx [hunk 1]
```diff
@@ -1,46 +1,84 @@
...
-            <Link to={`/getpostbyid/${id}`} className="btn btn-primary">Read more</Link>
-          </article>
-          <hr />
-      </>
-    );
-  };
-  
-  export default BlogPost;
+const BlogPost = ({ id, title, image, content, showTrimmedPost}) => {
+  console.log("ShowTrimmedPost ~~~~~~~~~~~>", showTrimmedPost)
```
Comment (sanjayguwaju) [inline src/components/BlogPost.jsx:6]: remove this line 6 console.log

File: src/components/BlogPost.jsx [hunk 2]
```diff
@@ -1,46 +1,84 @@
...
+          />
+          <figcaption className="figure-caption small">
+            <p className="mb-0">
+              <i className="fa fa-camera" aria-hidden="true"></i>
+              <span className="sr-only">Photo by:</span> Artist Name
+            </p>
+          </figcaption>
+        </figure>
+        {/* {console.log("kkkkkkkkkkkkkkkk",summary)} */}
+        {console.log("kkkkkkkkkkkkkkkk",showTrimmedPost)}
```
Comment (sanjayguwaju) [inline src/components/BlogPost.jsx]: remove the conosole.log

File: src/pages/Homepage.jsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ import BlogPost from '../components/BlogPost'
 
 const Homepage = () => {
     const [posts, setPost] = useState([]);
+    const [showTrimmedPost, setshowTrimmedPost] = useState(true);
+    // console.log("showTrimmedPost ~~~~~~~~~~~>",showTrimmedPost)
```
Comment (sanjayguwaju) [inline src/pages/Homepage.jsx]: remove this comment

**Review by sanjayguwaju — CHANGES_REQUESTED** [review]
please made the changes

File: src/components/BlogPost.jsx (hunk 1 above)
Comment (Kushalbaj) [reply src/components/BlogPost.jsx:6]: REMOVED

File: src/components/BlogPost.jsx (hunk 2 above)
Comment (Kushalbaj) [reply src/components/BlogPost.jsx]: REMOVED

File: src/pages/Homepage.jsx (hunk 3 above)
Comment (Kushalbaj) [reply src/pages/Homepage.jsx]: REMOVED

**Review by sanjayguwaju — APPROVED** [review]

**Pull request closed without merging**